core/amber/src/test/scala/edu/uci/ics/amber/engine/e2e/TestOperators.scala

```diff
@@ -21,6 +22,17 @@ object TestOperators {
     getCsvScanOpDesc("src/test/resources/CountrySalesDataSmall.csv", header = true)
   }
 
+  def smallJSONLScanOpDesc(): JSONLScanSourceOpDesc = {
+    getJSONLScanOpDesc("src/test/resources/100.jsonl")
+  }
+
+  def largeFlattenJSONLScanOpDesc(): JSONLScanSourceOpDesc = {
+    getJSONLScanOpDesc("src/test/resources/10000.jsonl", flatten = true)
```

Comment:
Are these 10000 tweets? Why do we even need such a huge number? Only small should suffice right?

Reply:
removed 10000